.env.example

```diff
@@ -2,4 +2,6 @@ REDIS_URL=redis://localhost:6379
 API_HOST_URL=https://api.helsingborg.se/event/json/wp/v2
 LANG_URL=https://api.helsingborg.se/event/json/pll/v1/languages
 HBG_GROUP_ID=
-CACHE_EXPIRATION_TIME=10
\ No newline at end of file
+CACHE_EXPIRATION_TIME=10
+CACHE_DISABLED=0
```

Comment:
Maybe change this to a positive-first variable to make logic more clear? `CACHE_ENABLED=1` is more readable than `CACHE_DISABLED=0` imo.

Reply:
Ah yes. That would require the ENV to be set on hosting though, while now it's more of an "optional". Not a real problem, it was just easier I figured :)

Reply:
Makes sense for backwards compatibility, I guess :) You can keep it as is 💯